pymc/gp/cov.py

```diff
@@ -47,26 +49,22 @@
 ]
 
 
-class Covariance:
-    r"""
-    Base class for all kernels/covariance functions.
+def _verify_scalar(value):
+    if (
```

Comment:
Why not call `at/np.squeeze` and capture the errors that both emit when the inputs are not allowed to be squeezed?